code/modules/paperwork/clipboard.dm

```diff
@@ -36,9 +38,13 @@
 	if(toppaper)
 		overlays += toppaper.icon_state
 		overlays += toppaper.overlays
+	else
+		var/obj/item/weapon/photo/Ph = locate(/obj/item/weapon/photo) in src
+		if(Ph)
+			overlays += image(Ph.icon)
 	if(haspen)
-		overlays += image(icon = icon, icon_state = "clipboard_pen")
-	overlays += image(icon = icon, icon_state = "clipboard_over")
+		overlays += "clipboard_pen"
```

Comment:
Don't do this, we're trying to convert things away from the overlays += string. Because Byond stores them in a weird way that fucks up bicon()